src/components/Entry/Summary/Wikipedia/index.tsx

```diff
@@ -113,7 +115,24 @@ const Wikipedia = ({ title, extract, thumbnail, data }: WikipediaProps) => {
                     ) : (
                       imageLink
                     )}
-                    <div>{article.caption}</div>
+                    <div>
+                      {captionWithLinksSplit.map((text, i) =>
+                        text.startsWith('[[') ? (
+                          <Link
+                            className={css('ext-link')}
+                            target="_blank"
+                            href={`${wikiLinkPrefix}${text.replaceAll(
+                              /[[\]]/g,
+                              '',
+                            )}`}
+                          >
+                            {text.replaceAll(/[[\]]/g, '')}
+                          </Link>
+                        ) : (
+                          <span key={i}>{text}</span>
+                        ),
```

Comment:
Do we really need to wrap each piece of text in a `<span>` element?

```suggestion
                        ) : text,
```

Reply:
This is more of a React requirement. When using a map to generate a list of elements, the element should have a key, which is important for dealing with re-renders.
That said, I forgot to put the Key in the `Link` element above. 🤦🏽